Sources/Sentry/include/SentryBaseIntegration.h

```diff
@@ -20,6 +20,7 @@ typedef NS_OPTIONS(NSUInteger, SentryIntegrationOption) {
     kIntegrationOptionEnableAutoBreadcrumbTracking = 1 << 12,
     kIntegrationOptionIsTracingEnabled = 1 << 13,
     kIntegrationOptionDebuggerNotAttached = 1 << 14,
+    kIntegrationOptionEnableCrashHandler = 1 << 16,
```

Comment:
I skipped 15 since that is added with https://github.com/getsentry/sentry-cocoa/pull/2044 :D